packaging/os/layman.py

```diff
@@ -248,7 +248,7 @@ def main():
         else:
             changed = uninstall_overlay(module, name)
 
-    except ModuleError, e:
+    except ModuleError as e:
```

Comment:
Why are you using `as e` here and in other places you are using `get_exception()` instead? Is the `as e` compatible with Python 2.4?